src/timeseriesflattener/feature_specs/single_specs.py

```diff
@@ -143,15 +182,29 @@ class PredictorSpec(BaseModel):
     feature_base_name: str
     aggregation_fn: AggregationFunType
     fallback: Union[float, int]
-    lookbehind_days: float
+    lookbehind_days: Union[float, Tuple[float, float]]
     prefix: str = "pred"
 
+    def __post_init__(self):
+        if isinstance(self.lookbehind_days, (float, int)):
+            self.lookbehind_days = (0, self.lookbehind_days)
+        check_lookperiod_tuple_is_valid(self.lookbehind_days)
```

Comment:
I'd move this into the lookahead_period property 👍

If the logic is identical, perhaps refactor out?